starknet_devnet/starknet_wrapper.py

```diff
@@ -511,6 +512,14 @@ async def __aexit__(
                     if not self.starknet_wrapper.config.blocks_on_demand:
                         await self.starknet_wrapper.generate_latest_block()
 
+                logger.info(
```

Comment:
I don't understand. Is this always logged?

Reply:
yes. Each transaction. But info level logs when verbose only.